bash_tab_complete

```diff
@@ -0,0 +1,33 @@
+# To use, in the Bash shell:
```

Comment:
I think we want to use the built-in capabilities that come with the picocli. I'll see if I can do some testing with this later:

https://picocli.info/autocomplete.html